include/enums.h

```diff
@@ -6584,13 +6584,12 @@ enum WindowStyles {
     WINDOW_STYLE_MAX    = 22,
 };
 
-#if VERSION_PAL
+#define LANGUAGE_DEFAULT 0
```

Comment:
Maybe LANGUAGE_EN is more appropriate
```suggestion
#define LANGUAGE_DEFAULT LANGUAGE_EN
```

Reply:
The idea here is that the first index of the arrays this often is used to index is the default language for that version (Chinese for iQue, since it only has one language. Japanese for JP, for the same reason)